### DISCUSSION TITLE
Missing ref in functional components

### DISCUSSION THREAD

**Discussion by diegofiore83** [material-table-core/core#119] [General]
👋 Hello 👋
I noticed that latest version of the library export **functional** components instead of **class** component. With this change the ref passed become undefined and solutions like the following are not anymore possible.

`<MTableToolbar {...props} ref={toolbarRef} />`

`toolbarRef.current.onSearchChange(search);`

Maybe considering something like this for future release could repristinate that functionality: https://levelup.gitconnected.com/react-refs-for-function-components-44f1a5a2332a

Thoughts?

**Comment by matthewoestreich** [comment]
This should be included now.
